### DISCUSSION TITLE
deploy on no SQL Database like mongoDB

### DISCUSSION THREAD

**Discussion by HusseinIbrahem** [spatie/laravel-activitylog#1361] [Ideas]
hi, the logs is very hivey and we need to save them in no SQL database for performance

**Comment by kde99** [comment]
You can just make a custom version of `\Spatie\Activitylog\Models\Activity` and replace it in the config file. Basically change the laravel Model in extends to the MongoDB Model class and should pretty much work. Apart from the age old bug of array/collection cast stuff are saved as strings, even with MongoDB.
